packages/flutter/test/material/time_picker_theme_test.dart

```diff
@@ -152,6 +153,21 @@ void main() {
           .merge(Typography.material2014().black.overline),
     );
 
+    final CustomPaint dialPaint = tester.widget(findDialPaint);
+    final dynamic dialPainter = dialPaint.painter;
```

Comment:
Can this be typed as CustomPainter?

Reply:
Unfortunately not, since I'm accessing properties from the private CustomPainter subclass, `_DialPainter`.